nucypher/blockchain/eth/interfaces.py

```diff
@@ -526,6 +526,12 @@ def build_contract_transaction(self,
             transaction_dict['gas'] = overestimation
             # TODO: What if we're going over the block limit? Not likely, but perhaps worth checking (NRN)
 
+        # Ensure higher gas price when sending a replacement tx
+        if use_pending_nonce:
+            self.log.debug(f"Gas price for this TX was increased from {transaction_dict['gasPrice']} "
+                           f"to {transaction_dict['gasPrice'] + 1} to prevent replacement TX from being stuck.")
+            transaction_dict['gasPrice'] += 1
```

Comment:
This doesn't work, as replacement TXs are dropped by peers if their price doesn't increment for some factor. Default in herb is 12.5%, IIRC.

But doing that it's also not a solution, because that can interfere with both the gas strategy and max gas limit set in the configuration. It's a very tricky problem.

The solution would be some sort of smart gas strategy, similar to what brownie has, but even those can end up work similar problems.